Project/Project/Src/KNN.cs

```diff
@@ -24,21 +26,57 @@ public void Train()
         public void Test(Dictionary<string, YoutubeChannel> testingDataList)
         {
             TestingDataList = testingDataList;
-
```

Comment:
galejai palikt tarpa

Reply:
galėjai

Reply:
cia matai refactor brnach kursim ir fixinsim namingus ir kitus sudukus

Reply:
nu galima viska iskart gerai padarytim, bet fck it